redskyctl/internal/commands/export/export.go

```diff
@@ -453,6 +463,8 @@ func (o *Options) getTrialDetails(ctx context.Context) (*trialDetails, error) {
 	result := &trialDetails{
 		Experiment:  experimentName.Name(),
 		Application: exp.Labels["application"],
+		Scenario:    exp.Labels["scenario"],
+		Objective:   exp.Labels["objective"],
```

Comment:
do we need to verify these keys exist first?

Reply:
Nope, in that case they will just be set to empty strings (I love Go's zero value maps) which will match the one and only scenario or objective. If there is no label and the application definition contains multiple choices, it will fail in get `GetScenario` or `GetObjective`.